nngt/plot/plt_networks.py

```diff
@@ -63,7 +63,7 @@ def draw_network(network, nsize="total-degree", ncolor="group", nshape="o",
                  threshold=0.5, decimate=None, spatial=True,
                  restrict_sources=None, restrict_targets=None,
                  show_environment=True, fast=False, size=(600,600), xlims=None,
-                 ylims=None, dpi=75, axis=None, show=False, **kwargs):
+                 ylims=None, dpi=75, axis=None, show=False, title="", save_graph=False,**kwargs):
```

Comment:
Add whitespace between ``save_graph=False,`` and ``**kwargs``.
Please respect PEP8 guidelines by avoiding lines that are longer that 80 characters (move ``save_graph`` and ``kwargs`` to a new line)